fedora/dmtcp.spec

```diff
@@ -1,17 +1,20 @@
 %{!?_pkgdocdir: %global _pkgdocdir %{_docdir}/%{name}-%{version}}
 
 Name:		dmtcp
-Version:	2.4
-Release:	0%{?dist}
+Version:	3.0.0
+Release:	0.1%{?dist}
```

Comment:
Please change this back to `0`. This is supposed to be an integer starting at `0` for each new release of the _same_ tarball.